packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/ChangePlan/PaymentMethod.tsx

```diff
@@ -0,0 +1,198 @@
+    },
+    rowBox: {
+      display: "flex",
+      justifyContent: "space-between"
+    },
+    pushRightBox: {
+      display: "flex",
+      flexDirection: "column",
+      alignItems: "flex-end",
+      flex: 1
+    },
+    buttons: {
+      display: "flex",
+      flexDirection: "row",
+      alignItems: "center",
+      justifyContent: "flex-end",
+      "& > *": {
+        marginLeft: constants.generalUnit
+      }
+    },
+    bottomSection: {
+      display: "flex",
+      flexDirection: "row",
+      justifyContent: "space-between",
+      alignItems: "center",
+      marginTop: constants.generalUnit * 4,
+      marginBottom: constants.generalUnit * 3
+    },
+    divider: {
+      margin: `${constants.generalUnit}px 0`
+    },
+    radioLabel: {
+      fontSize: 14
+    },
+    textButton: {
+      color: palette.primary.background,
+      cursor: "pointer"
+    },
+    linkButton: {
+      textDecoration: "underline",
+      cursor: "pointer"
+    },
+    addCardWrapper: {
+      padding: `${constants.generalUnit * 2}px 0px`
+    },
+    footer: {
+      marginTop: constants.generalUnit * 2
+    }
+  })
+)
+
+interface IPaymentMethodProps {
+  onClose: () => void
+  goToSelectPlan: () => void
+  goToPlanDetails: () => void
+  onSelectPaymentMethod: () => void
+}
+
+const PlanDetails = ({ onClose, goToSelectPlan, goToPlanDetails, onSelectPaymentMethod }: IPaymentMethodProps) => {
+  const classes = useStyles()
+  const [paymentMethod, setPaymentMethod] = useState<"creditCard" | "crypto" | undefined>()
+  const [view, setView] = useState<"selectPaymentMethod" | "addCard">("selectPaymentMethod")
+  const { defaultCard } = useBilling()
+
+  useEffect(() => {
+    if (defaultCard) {
+      setPaymentMethod("creditCard")
+    }
+  }, [defaultCard])
+
+  return (
+    <article className={classes.root}>
+      <Breadcrumb
+        crumbs={[{
+          text: t`Change plan`,
+          onClick: goToSelectPlan
+        }, {
+          text: t`Plan details`,
+          onClick: goToPlanDetails
+        }, {
+          text: t`Payment method`
+        }]}
+        hideHome={true}
+      />
+      <Typography variant="h5"
+        component="h4"
+        className={classes.heading}
+      >
+        <Trans>Select payment method</Trans>
+      </Typography>
+      <Typography variant="body1"
+        component="p"
+        className={classes.subHeading}
+      >
+        {view === "addCard" && <Trans>This card will become your default payment method</Trans>}
+      </Typography>
+      <Divider className={classes.divider} />
+      {view === "selectPaymentMethod"
+        ? <>
+          <div className={classes.rowBox}>
+            <RadioInput
+              label={defaultCard ? `•••• •••• •••• ${defaultCard.last_four_digit}` : "Credit card" }
+              value="creditCard"
+              onChange={() => setPaymentMethod("creditCard")}
+              checked={paymentMethod === "creditCard"}
+              labelClassName={classes.radioLabel}
```

Comment:
This should be disabled if there's no default card. I was able to change plan without card.
```suggestion
              labelClassName={classes.radioLabel}
              disabled={!defaultCard}
```